src/main/java/SchemaExportter.java

```diff
@@ -0,0 +1,49 @@
+
+import org.hibernate.boot.MetadataSources;
+import org.hibernate.boot.internal.MetadataBuilderImpl;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.tool.hbm2ddl.SchemaExport;
+
+import fi.otavanopisto.kuntaapi.server.persistence.model.Identifier;
+
+public class SchemaExportter {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)